.github/workflows/deploy.yml

```diff
@@ -3,7 +3,7 @@ on:
   push:
     branches: [ main ]
 env:
-  DOCKER_IMAGE: docker.pkg.github.com/${{ github.repository }}/${{ github.repository }}
+  DOCKER_IMAGE: docker.pkg.github.com/${{ github.repository }}/divergent-console
```

Comment:
what does github.repository evaluate to?
`docker.pkg.github.com/divergent-console/divergent-console`

Reply:
`{{ github.repository }}` -> `kibalabs/divergent-console`
and ye the whole path is `docker.pkg.github.com/kibalabs/divergent-console/divergent-console` as you said. it will show up here: https://github.com/orgs/kibalabs/packages

Reply:
(you can see there why the name has to be used in full again - basically the text after the last forward slash is what will appear in that list. Its super wierd cos kinda makes it seem like the repository is not really needed but they might use it somewhere internally